doc/source/whats_new.rst

```diff
@@ -6,6 +6,11 @@
  What's New
 ============
 
+v3.0.0 (tba)
+============
+- Drop support for Python 3.6, 3.7 and 3.8
+- Wheels for Python 3.9, 3.10, 3.11 and 3.12
```

Comment:
Dropping support for Python versions is a breaking change I guess and requires a major version bump

Reply:
(We may not forget the set the date here)